src/index.js

```diff
@@ -131,18 +150,18 @@ export default function({ types: t }) {
                 }
                 const methodName = clsMethod.node.key.name;
 
-                const protoProp  = getProtoProp(defineCall);
-                const isOverride = protoProp && protoProp.key.name === 'override';
-                const protoRef   = getProtoRef(protoProp);
+                const protoProp = getProtoProp({ defineCall, overrideCall });
+                const isDefineOverride = protoProp && protoProp.key && protoProp.key.name === 'override';
+                const protoRef = getProtoRef(protoProp);
+
                 let methodRef = buildMethodRef(protoRef, methodName);
-                if (isOverride) {
-                    methodRef = getOverrideMethodRef(methodRef, defineCall);
+                if (isDefineOverride) {
+                    methodRef = getDefineOverrideMethodRef(methodRef, defineCall);
                 }
 
                 const args = path.node.arguments;
                 path.replaceWith(buildReplacement(methodRef, args));
             }
         }
     };
-};
```

Comment:
All files deliberately end in a trailing newline, could this change be reverted?

Reply:
I readded the trailing newlines.